scripts/ts/common/ticket/jira/types.ts

```diff
@@ -0,0 +1,53 @@
+import * as t from "io-ts";
+
+const JiraIssueType = t.union([
```

Comment:
pretty the same but the io-ts author suggest to use `keyof` for keys uniqueness advantage
https://github.com/gcanti/io-ts/issues/128#issuecomment-365172709

Reply:
Nice find! Changed in 01ef487